packages/js/src/elementor/initializers/pluggable.js

```diff
@@ -11,8 +10,12 @@ let pluggable = null;
  */
 const getPluggable = () => {
 	if ( pluggable === null ) {
+		// Use the initialized pluggable plugin in post-scrapper if it's available.
+		// Otherwise, initiate a new Pluggable plugin.
 		const refresh = dispatch( "yoast-seo/editor" ).runAnalysis;
-		pluggable = new Pluggable( refresh );
+		pluggable = window.YoastSEO.app && window.YoastSEO.app.pluggable
+			? window.YoastSEO.app.pluggable
+			: new Pluggable( refresh );
```

Comment:
Pluggable is not available in `window.YoastSEO` in Elementor. So, in this case, we still need to create a new instance of Pluggable plugin.